File: from-2901-to-3000/2938-separate-black-and-white-balls/solution.py
class Solution:
    def minimumSteps(self, s: str) -> int:
        result = 0
        zeros = 0
        for i in range(len(s) - 1, -1, -1):
            if s[i] == '0':
                zeros += 1
            else:
                result += zeros

        return result

File: from-2901-to-3000/2938-separate-black-and-white-balls/solution.go
func minimumSteps(s string) int64 {
    var result, zeros int64 = 0, 0
    for i := len(s) - 1; i >= 0; i -- {
        if s[i] == '0' {
            zeros += 1
        } else {
            result += zeros
        }
    }
    return result
}